src/shared/ui/navigation/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,201 @@
+"use client";
+
+import * as DropdownMenuPrimitive from "@radix-ui/react-dropdown-menu";
+import { HiChevronRight } from "react-icons/hi2";
+import { IoMdCheckmark } from "react-icons/io";
+import { RxDotFilled } from "react-icons/rx";
+
+import * as React from "react";
+
+import { twJoin } from "tailwind-merge";
+
+export const DropdownMenu = DropdownMenuPrimitive.Root;
+
+export const DropdownMenuTrigger = DropdownMenuPrimitive.Trigger;
+export const DropdownMenuGroup = DropdownMenuPrimitive.Group;
+
+export const DropdownMenuPortal = DropdownMenuPrimitive.Portal;
+
+export const DropdownMenuSub = DropdownMenuPrimitive.Sub;
+
+export const DropdownMenuRadioGroup = DropdownMenuPrimitive.RadioGroup;
+
+export const DropdownMenuSubTrigger = React.forwardRef<
+  React.ElementRef<typeof DropdownMenuPrimitive.SubTrigger>,
+  React.ComponentPropsWithoutRef<typeof DropdownMenuPrimitive.SubTrigger> & {
+    inset?: boolean;
+  }
+>(function DropdownMenuSubTrigger(
+  { className, inset, children, ...props },
+  ref,
+) {
+  return (
+    <DropdownMenuPrimitive.SubTrigger
+      ref={ref}
+      className={twJoin(
+        "focus:bg-accent data-[state=open]:bg-accent flex cursor-default select-none items-center rounded-sm px-2 py-1.5 text-sm outline-none",
```

Comment:
I think `focus:bg-accent data-[state=open]:bg-accent` are useless here?
Should we use `bg-background-color-bright-accent` instead?

Can we use our css variables instead of `rounded-sm px-2 py-1.5 text-sm`?

Reply:
YAGNI !

`<DropdownMenuPrimitive.SubTrigger>` has been removed and will return as needed !